portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTestACSide.testcase

```diff
@@ -290,7 +290,13 @@ definition {
 		}
 
 		task ("Delete the test in AC") {
-			ABTest.deleteABTestACSide();
+			ACTests.deleteABTestACSide();
+
+			AssertTextEquals.assertPartialText(
+			locator1 = "Modal#MODAL_CONTENT",
+			value1 = "Are you sure you want to delete this test? You will permanently lose all test data and results collected from this test. You will not be able to run this test. You will not be able to undo this operation.");
```

Comment:
Hey @manuferreira, I think this test will fail since the modal will not appear at this moment. When you use `ACTests.deleteABTestACSide`, then the test will click on the delete button and then on the delete confirmation button which is exactly in the modal, so when you check the modal, it will not appear on the screen

Reply:
Since we only have this test to assert the deletion message on the AC side. What I will do is:
Do not use this deleteABTestACSide macro, because like you mentioned earlier, it's not good to put asserts in a macro whose goal is not to assert but to do something else (like the case of this macro).
So I will remove the macro of this specific test and put the code to click on the buttons inside the test.

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/baad52eb2adcf50e80d1a475591fe4facd1d3ae2